src/components/TransformDelimiterSeparatedValues.tsx

```diff
@@ -5,6 +5,7 @@ import { applyReshapeTransformationArray } from '../utilities.ts';
 
 interface Props extends IOutputComponentProps {
 	delimiter: string;
+	setIsDisabled:React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
```suggestion
	setIsDisabled: StateSetter<boolean>;
```

See `StateSetter`, an alias that exists in this project for this exact use. Make sure you import this too

https://github.com/allejo/json-reshaper/blob/e06ad7911215b4a7c269ec2dd9c294be9fe8ed8d/src/contracts.ts#L41